tests/samplers_tests/test_grid.py

```diff
@@ -37,7 +37,7 @@ def objective(trial: Trial) -> float:
     search_space = {
         "b": np.arange(-0.1, 0.1, 0.05),
         "c": ("x", "y", None, 1, 2.0),
-        "d": [-0.5, 0.5],
```

Comment:
These grid values are out-of-range in the corresponding suggest method, `trial.suggest_float("d", -5, 5, step=1)`.